inst/template/report_fr.epub3

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="https://www.idpf.org/2007/ops"$if(lang)$ xml:lang="$lang$"$endif$>
+<head>
+  <meta charset="utf-8" />
+  <meta name="generator" content="pandoc" />
+  <title>$pagetitle$</title>
+  <style type="text/css">
+      code{white-space: pre-wrap;}
+      span.smallcaps{font-variant: small-caps;}
+      span.underline{text-decoration: underline;}
+      div.column{display: inline-block; vertical-align: top; width: 50%;}
+$if(quotes)$
+      q { quotes: "“" "”" "‘" "’"; }
+$endif$
+  </style>
+$if(highlighting-css)$
+  <style type="text/css">
+$highlighting-css$
+  </style>
+$endif$
+$for(css)$
+  <link rel="stylesheet" type="text/css" href="$css$" />
+$endfor$
+$for(header-includes)$
+  $header-includes$
+$endfor$
+</head>
+<body$if(coverpage)$ id="cover"$endif$$if(body-type)$ epub:type="$body-type$"$endif$>
+$if(titlepage)$
+<section epub:type="titlepage" class="titlepage">
+  <h1 class="title">$title$</h1>
+  $if(subtitle)$
+  <h2 class="subtitle">$subtitle$</h2>
+  $endif$
+  $for(author)$
+    $if(author.name)$
+      <h3 class="author">$if(author.firstname)$$author.firstname$ $endif$$author.name$
+      $if(author.orcid)$
+        <a href="https://orcid.org/$author.orcid$">
+        <img alt="ORCID logo" src="https://info.orcid.org/wp-content/uploads/2019/11/orcid_16x16.png" width="16" height="16" />
+        $author.orcid$
+        </a>
+      $endif$
+      </h3>
+      $if(author.affiliation)$
+        <address class="author_afil">
+        $author.affiliation$
+        $if(author.email)$
+          <br><a class="author_email" href="mailto:#">$author.email$</a>
+        $endif$
+        </address>
+      $else$
+        $if(author.email)$
+          <address class="author_afil">
+          <a class="author_email" href="mailto:#">$author.email$</a>
+          </address>
+        $endif$
+      $endif$
+    $else$
+      <h3 class="author">$author$</h3>
+    $endif$
+  $endfor$
+  $if(date)$
+  <p class="date"><em>$date$</em></p>
+  $endif$
+  $if(doi)$
+  <h4><a class="doi" href="#">$doi$</a></h4>
+  $endif$
+  $if(ordernr)$
+  <h4 class ="ordernr">$ordernr$</h4>
+  $endif$
+  <h1>Colophon</h1>
+  $if(reviewer)$
+    <p><b>Reviewers:</b><br>
```

Comment:
```suggestion
    <p><b>Réviseurs:</b><br>
```
What do you think @cecileherr?

Reply:
I'll leave this one to @cecileherr

Reply:
I would say you could use 'reviewers' (not even translated thus), 'relecteurs' (comes from 'revue à comité de lecture') or 'réviseurs' for this. Ok thus for 'réviseurs'

Reply:
@ThierryO Which one do you prefer?

Reply:
What would our Walloon friends use @cecileherr? I"m fine with reviewers is that is the common term to use in Belgium.